control_plane/src/local_env.rs

```diff
@@ -562,6 +562,10 @@ impl LocalEnv {
             fs::create_dir_all(SafekeeperNode::datadir_path_by_id(self, safekeeper.id))?;
         }
 
+        for ps in &self.pageservers {
+            fs::create_dir(self.pageserver_data_dir(ps.id))?;
```

Comment:
use `create_dir_all` to avoid throwing an error on existing dir?

Reply:
Error is desired. This code path is only taken during init.